### DISCUSSION TITLE
Update to latest version 0.31.2 of CommonMark

### DISCUSSION THREAD

**Discussion by hartmanjan1970** [bsorrentino/maven-confluence-plugin#299] [General]
There is a way newer version of CommonMark.
https://spec.commonmark.org/0.31.2/

**Comment by bsorrentino** [comment]
Hi @hartmanjan1970 

I'm working on an upgrade of plugin toward java version 11. I'll plan to upgrade commonmark. 
What are the main features of this new commonmark version  that you want highlight ?

**Reply by hartmanjan1970** [reply]
Hi,
i was looking for tables, how to add an table in an `.md` file so i get a table in confuence.
Then i saw that the plugin was behind with the current CommonMark version.
I tried to make a pull request with this update, but locally i did not succeed in making the source compile.

We are very happily making use of this plugin with so i made this item with update request.

thanks for the quick response.

Jan

**Reply by bsorrentino** [reply]
Currently plugin support markdown table (take look [here](http://bsorrentino.github.io/maven-confluence-plugin/markdown_guide.html)). Do you mean something special?

**Reply by hartmanjan1970** [reply]
yes, exactly, that was what I was looking for.
my problem was that the table is in markdown format used in a `asyncApi. yml` file which converts to a  `.md` and this file is published to confluence.
my problem was solved by adding an complete empty line after every row of the table.
see example below. 

```
| Tables        | Are           | Cool  |

| ------------- |:-------------:| -----:|

| col 3 is      | right-aligned | $1600 |

| col 2 is      | centered      |   $12 |

| zebra stripes | are neat      |    $1 |

```

**Comment by bsorrentino** [comment]
Glad to hear that you solved the issue

However the release `0.31.2` is not available yet in [commonmark-java](https://github.com/commonmark/commonmark-java)
